README.tfdoc.hcl

```diff
@@ -1057,18 +1057,41 @@ section {
             This map allows you to create and manage secrets for repositories in your organization.
             Each element in the map is considered a secret to be managed, being the key map the secret name and the value the corresponding secret in plain text:
 
-            ```
+            When applied, a secret with the given key and value will be created in the repositories.
+
+            The value of the secrets must be given in plain text, GitHub provider is in charge of encrypting it.
+
+            **Attention:** You might want to get secrets via a data source from a secure vault and not add them in plain text to your source files, so you do not commit plaintext secrets into the git repository managing your GitHub account.
```

Comment:
this is true but we should rather discourage from using plaintext and encourage encryption. from the provider docs:

> You should treat state as sensitive always. It is also advised that you do not store plaintext values in your code but rather populate the encrypted_value using fields from a resource, data source or variable as, while encrypted in state, these will be easily accessible in your code. See below for an example of this abstraction.